srcDoc/tutorials/performance_validation.rst

```diff
@@ -0,0 +1,642 @@
+is not saturated, this can be done again with **Perftest**, doing a "Latency Test". In order
+to do that, you only need to add `-latencyTest` to the previous command line parameters of the
+publisher side.
+
+* **Publisher side**
+
+    .. code::
+
+        for DATALEN in 32 64 128 256 512 1024 2048 4096 8192 16384 32768 63000; do
+            bin/armv6vfphLinux3.xgcc4.7.2/release/perftest_cpp -pub -peer 10.45.3.119 -nic eth0 -raw -noPrint -exec 20 -datalen $DATALEN -latencyTest;
+        done
+
+* **Subscriber side**
+
+    .. code::
+
+        for DATALEN in 32 64 128 256 512 1024 2048 4096 8192 16384 32768 63000; do
+            bin/armv6vfphLinux3.xgcc4.7.2/release/perftest_cpp -sub -peer 10.45.3.120 -nic eth0 -raw -noPrint -datalen $DATALEN;
+        done
+
+Remember that in this case we are interested in the latency results, not in the
+throughput results (we are doing a ping-pong test, so we cannot expect high throguhput),
+therefore we need to look to the results displayed in the publisher side.
+
+Latency Results -- RAW Transport (UDPv4)
+::::::::::::::::::::::::::::::::::::::::
+
+    .. csv-table::
+        :align: center
+        :header-rows: 1
+
+        "Size", "Ave (us)", "Std (us)", "Min (us)", "Max (us)", "50% (us)", "90% (us)", "99% (us)", "99.99% (us)", "99.9999% (us)"
+        32,357,77.7,310,6094,355,371,470,5436,6094
+        64,370,76.5,305,3935,365,387,491,3693,3935
+        128,386,88.3,318,6573,381,403,512,5549,6573
+        256,419,82.0,360,6451,416,438,546,4810,6451
+        512,485,72.5,435,5913,479,503,610,4571,5913
+        1024,608,96.5,545,6507,602,633,757,6435,6507
+        2048,809,102.2,736,5605,797,845,994,5318,5605
+        4096,1027,120.2,952,8083,1015,1058,1196,8083,8083
+        8192,1412,106.1,1325,5969,1400,1456,1608,5969,5969
+        16384,2107,222.5,1931,9573,2096,2153,2338,9573,9573
+        32768,3693,223.2,3477,8656,3696,3768,4046,8656,8656
+        63000,6601,212.9,6424,10706,6595,6752,7002,10706,10706
+
+RTI Connext DDS Professional (UDPv4)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Throughput Test
+---------------
+
+The idea is the same as we did in the previous test, get the maximum throughput we can
+achieve, but this time we will use our middleware to test with (**RTI Connext DDS Professional 6.0.0**)
+
+Then, the command line parameters are going to be quite similar:
+
+* **Publisher side**
+
+    .. code::
+
+        bin/armv6vfphLinux3.xgcc4.7.2/release/perftest_cpp -pub -nic eth0 -noPrint -exec 20 -scan -batchSize 0
+
+* **Subscriber side**
+
+    .. code::
+
+        bin/armv6vfphLinux3.xgcc4.7.2/release/perftest_cpp -sub -nic eth0 -noPrint;
+
+Notice that now we removed the `-raw` parameter, and that we do not need the *for loop* anymore, since
+**Perftest** for **Connext DDS** support the use of the `-scan` parameter. Also notice that we are using
+`-batchSize 0`. We will also later using batching. Lastly, we also removed the `-peer` parameter, the reason
```

Comment:
Replace: 

We will also later using batching

With:

We will also use batching later

Reply:
Done